src/index.js

```diff
@@ -32,19 +35,23 @@ const build = async (functionsPath, publishPath) => {
 
   setupHeaders(publishPath)
 
+  if (nextRoot !== oldCwd) {
+    process.chdir(oldCwd)
+  }
+
```

Comment:
wait. whats going on here with l22-25 and l38-41 🤔

Reply:
This is because of the way NoN assumes it's running in the root of the Next site. We chdir into the root before running it, then move back to wherever we were before

Reply:
oh duh ok. maybe a comment for someone like me? :D 🙏